stdlib/list.ml

```diff
@@ -283,6 +283,17 @@ let partition p l =
   | x :: l -> if p x then part (x :: yes) no l else part yes (x :: no) l in
   part [] [] l
 
+let partition_map p l =
+  let rec part yes no = function
```

Comment:
It would be a little clearer to call `yes` and `no` `left` and `right`.

Reply:
(done)